Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -121,8 +130,8 @@ public void run() {
         mAdConfig.setAdSize(vungleAdSize);
         sVungleRouter.addRouterListener(mPlacementId, mVungleRouterListener);
 
-        VungleMediationConfiguration.adConfigWithExtras(mAdConfig, extras);
-        if (VungleMediationConfiguration.isStartMutedNotConfigured(extras)) {
+        VungleMediationConfiguration.adConfigWithLocalExtras(mAdConfig, localExtras);
+        if (VungleMediationConfiguration.isStartMutedNotConfiguredLocal(localExtras)) {
```

Comment:
There's no difference between `isStartMutedNotConfiguredLocal (localExtras)` and `isStartMutedNotConfigured(localExtras)`. Suggestion to reuse it.
```
if (VungleMediationConfiguration.isStartMutedNotConfigured(localExtras)) {
```

Reply:
they have different parameter type. I'll change type for isStartMutedNotConfigured's parameter to Map<String, Object>, and remove isStartMutedNotConfiguredLocal